_activities/differentiablecomputing.md

```diff
@@ -0,0 +1,23 @@
+## Why make things differentiable?
+
+When we write a program to do some physics, that program will likely have some free parameters. These could be as simple as the position of a cut, or as complicated as the parameters of a neural network. In either case, we want these parameters to be optimized such that we get the best possible result, whatever that may be. In the case of a HEP analysis, for instance, this could correspond to having the highest sensitivity to new phenomena. 
+
+So, how exactly do we get there?
+
+It turns out that we can do exactly what we do when we train a neural network -- update the parameters using gradient descent. This is where the _differentiable_ part
+comes in: we can only optimize with respect to an objective that has a tractable gradient, since that's how we tell our parameters which direction to go in order to improve. Since there are usually many steps between the use of the parameters and the program output, it follows that every operation in-between them must also be differentiable. This lets us track exactly how changing the parameters affects the final result, and allows us to optimize our physics calculations _end-to-end_.
+
+## How do we do it?
+We can make this possible if we keep track of the derivatives of each step (i.e. line of code) of the program with respect to its inputs. While that may sound like a harsh requirement, this is made pretty simple thanks to the magic of automatic differentiation. Of course, this is not without its caveats, as not all lines of code are necessarily differentiable. In particular, common operations in HEP like binning a set of data or making a cut do not vary smoothly with respect to their inputs. That’s why there’s an ongoing effort within this activity to provide drop-in differentiable replacements for common operations, as well as more sophisticated differentiable ‘blocks’, such as statistical model building using [pyhf](https://github.com/scikit-hep/pyhf), or inference using the profile likelihood as a test statistic.
+
+## How do I get involved?
+
+Efforts in this direction are currently pivoting around the [gradHEP](gradhep.github.io) organisation on GitHub. If you’d like to get involved, feel free to [express interest on Gitter](https://gitter.im/gradhep/community?source=orgpage), or [raise an issue on our ‘center’ repo](https://github.com/gradhep/center/issues) if you have some big picture thoughts to share.
```

Comment:
For gradHEP you need an absolute URL, like `https://gradhep.github.io`

Reply:
fixed